eeps/0022-rename-cross-attrs.md

```diff
@@ -75,10 +75,10 @@ stdenv.mkDerivation {
   ...
 
   # CMake is used at buildtime, to produce code for the host platform
-  buildtimeDeps = [ cmake ];
+  commands = [ cmake ];
```

Comment:
Hmm, a static binary could have build time deps which are libraries

Reply:
Do you have an example?

Pretty sure the `.a`'s are still targeting `hostTarget` which would be a buildInputs/library